src/WebJobs.Extensions.DurableTask/DurableOrchestrationContext.cs

```diff
@@ -234,9 +235,28 @@ public Task CallFunctionAsync(string functionName, params object[] parameters)
 
             // TODO: Support for versioning
             string version = DefaultVersion;
-            this.config.AssertActivityExists(functionName, version);
+            
+            Task<TResult> callTask = null;
+            string activityFunctions = null;
```

Comment:
> = null [](start = 36, length = 7)

nit: don't initialize to null unless the compiler requires it. By not initializing them, we allow the compiler's flow analysis to validate that these local variables are assigned the way we expect them to be.